src/Database/Database.cpp

```diff
@@ -68,7 +68,8 @@ namespace CollabVM {
 									   make_column("QMPAddress", &VMSettings::QMPAddress),
 									   make_column("QMPPort", &VMSettings::QMPPort),
 									   make_column("QEMUCmd", &VMSettings::QEMUCmd),
-									   make_column("QEMUSnapshotMode", &VMSettings::QEMUSnapshotMode))
+									   make_column("QEMUSnapshotMode", &VMSettings::QEMUSnapshotMode),
```

Comment:
I'm pretty sure that this *may* cause DB schemas to be thrown out entirely. Regardless, for transitional purposes, it should allowed to be null (especially for no-password.) sqlite_orm does have this functionality, however VMSettings will have to be changed to make VMPassword a unique_ptr